spec/registry.md

```diff
@@ -255,3 +255,502 @@ which only expects the `accord` option:
 > .input {$object :noun case=nominative}
 > {{You see {$color :adjective article=indefinite accord=$object} {$object}!}}
 >```
+
+# Default Registry for LDML45
+
+This section describes the functions which each implementation MUST provide to be conformant with
+the LDML45 Technical Preview of MessageFormat v2.
+
+## String Value Selection and Formatting
+
+(Pending)
+
+## Numeric Value Selection and Formatting
+
+### Functions
+
+The following functions use numeric selection:
+
+The function `:number` is the default selector for numeric values.
+
+The function `:integer` provides a reduced set of options for selecting
+and formatting numeric values as integers.
+
+### Operands
+
+The _operand_ of a number function is either an implementation-defined type or
+a literal that matches the `number-literal` production in the [ABNF](/main/spec/message.abnf).
```

Comment:
As before, including `/main` in these links is invalid. So the same applies to all of them:
```suggestion
a literal that matches the `number-literal` production in the [ABNF](/spec/message.abnf).
```